Gemfile.lock

```diff
@@ -277,13 +277,14 @@ GEM
     omniauth-twitter (1.4.0)
       omniauth-oauth (~> 1.1)
       rack
+    parallel (1.12.0)
     parser (2.4.0.0)
       ast (~> 2.2)
-    pg (0.20.0)
+    pg (0.21.0)
```

Comment:
Please don't perform `bundle update` since some dependency versions (e.g. `pg`) are currently broken 😞

Reply:
About the changes, I went with the user's suggestion, but yes, showing the email address on hover would make sense.
I will see what I can do later today. 👍

Reply:
About the changes, I went with the user's suggestion, but yes, showing the email address on hover would make sense.
I will see what I can do later today. 👍